pkg/build/builder/sti.go

```diff
@@ -222,7 +222,15 @@ func (s *S2IBuilder) Build() error {
 		return errors.New(buffer.String())
 	}
 
-	// If DockerCfgPath is provided in api.Config, then attempt to read the the
+	// Resolve the runtime image reference to a docker pull spec if it has been
+	// specified and set the fields for s2i builder.
```

Comment:
wrong comment :) will fix. more like "resolve docker auth for runtime and builder" what i'm referring to here already happened in generator.go.